testing/test_pytester.py

```diff
@@ -323,9 +325,9 @@ def test_remove_added(self) -> None:
         snapshot.restore()
         assert sys.modules == original
 
-    def test_add_removed(self, monkeypatch) -> None:
+    def test_add_removed(self, monkeypatch: MonkeyPatch) -> None:
         assert self.key not in sys.modules
-        monkeypatch.setitem(sys.modules, self.key, "something")
+        monkeypatch.setitem(sys.modules, self.key, self.mod)
```

Comment:
had to cast into `ModuleType` to avoid `Cannot infer type argument 2 of "setitem" of "MonkeyPatch"` mypy error.

Reply:
:+1: 

It would be a bit more straightforward to read if the `ModuleType("something")` was inlined directly instead of using the `self.mod` field. It's repetitious but that's not so bad in tests IMO.